pkg/kubelet/kuberuntime/kuberuntime_container_linux.go

```diff
@@ -352,12 +351,12 @@ var swapControllerAvailable = func() bool {
 		p := "/sys/fs/cgroup/memory/memory.memsw.limit_in_bytes"
 		if isCgroup2UnifiedMode() {
 			// memory.swap.max does not exist in the cgroup root, so we check /sys/fs/cgroup/<SELF>/memory.swap.max
-			_, unified, err := cgroups.ParseCgroupFileUnified("/proc/self/cgroup")
+			cm, err := libcontainercgroups.ParseCgroupFile("/proc/self/cgroup")
 			if err != nil {
 				klog.V(5).ErrorS(fmt.Errorf("failed to parse /proc/self/cgroup: %w", err), warn)
 				return
 			}
-			p = filepath.Join("/sys/fs/cgroup", unified, "memory.swap.max")
+			p = filepath.Join("/sys/fs/cgroup", cm[""], "memory.swap.max")
```

Comment:
This seems a bit esoteric..

Can you explain why we are grabbing cm[""] here?

Reply:
(explain by adding a comment into code)

Reply:
The answer is a click away (in my $EDITOR, at least) -- take a look at the [ParseCgroupFile docs](https://pkg.go.dev/github.com/opencontainers/runc/libcontainer/cgroups#ParseCgroupFile), it says:

> Note that for cgroup v2 unified hierarchy, there are no per-controller cgroup paths, so the resulting map will have a single element where the key is empty string ("") and the value is the cgroup path the <pid> is in.

Nevertheless, I've added the comment.